### DISCUSSION TITLE
Change Pastebin service

### DISCUSSION THREAD

**Discussion by Andre601** [Ghost-chu/QuickShop-Reremake#1530] [Ideas]
It seems like that https://paste.ubuntu.com/ now requires you to login in order to see generated pastes, as I was greeted with a login request when loading a paste URL generated.

![image](https://user-images.githubusercontent.com/11576465/139560454-05667916-483d-4bdf-8511-554dfbfc8866.png)

Unless this is intentional (i.e. for protecting possibly sensitive data) should you consider to look into another paste service to use.

My go-to recommendation would be https://paste.helpch.at as they have a fairly high text limit (if even one at all) and no expiry time.
Another alternative could be https://paste.gg which is like github gists but with support for anonamous posts.

**Comment by Ghost-chu** [comment]
https://paste.helpch.at already added into the plugin as first try provider.
paste.gg also awesome, I will add it later.


Ubuntu Pastebin can accept larger content entity, so that's is why I still keep it in the plugin.

**Reply by Andre601** [reply]
Yeah, but as mentioned do they apparently require an account to see the content which is just a no-go for most people.

**Reply by Ghost-chu** [reply]
> Yeah, but as mentioned do they apparently require an account to see the content which is just a no-go for most people.

paste.gg require use browser to do JavaScript challenge, so I don't think it can be add into QuickShop paster

**Reply by Andre601** [reply]
It has an API so I doubt that one requires to pass a challenge

**Comment by Ghost-chu** [comment]
will check that before release next build :)












Ghost_chu
***@***.***


---Original---
From: ***@***.***&gt;
Date: Mon, Nov 1, 2021 22:06 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Ghost-chu/QuickShop-Reremake] Change Pastebin service (Discussion #1530)




 
It has an API so I doubt that one requires to pass a challenge

**Reply by Andre601** [reply]
https://github.com/ascclemens/paste/blob/master/api.md
